components/nds-accordion/src/AccordionGroup.tsx

```diff
@@ -48,7 +48,7 @@ export const AccordionGroup: FC<AccordionGroupProps> = ({
 					<Toggle isOpen={isGroupOpen}>{toggleText(isGroupOpen)}</Toggle>
 				</button>
 			) : null}
-			{children}
+			<div data-id="accordion-group">{children}</div>
```

Comment:
Pls can we add a class to these too so we're not using data attributes for styles